include/grpc++/impl/call.h

```diff
@@ -90,10 +93,12 @@ class CallOpBuffer : public CompletionQueueTag {
   grpc_metadata_array recv_initial_metadata_arr_;
   // Send message
   const grpc::protobuf::Message *send_message_;
-  grpc_byte_buffer *send_message_buf_;
+  const ByteBuffer *send_message_buffer_;
+  grpc_byte_buffer *send_buf_;
```

Comment:
I also want to ask the same question as Feng's comment. Shouldn't this be a single thing and maybe use the Buffer() method of the ByteBuffer if you ever need to extract the underlying grpc_byte_buffer?